src/harness/harnessUtils.ts

```diff
@@ -218,7 +218,7 @@ namespace Utils {
                     case "symbolCount":
                     case "identifierCount":
                     case "scriptSnapshot":
-                        // Blacklist of items we never put in the baseline file.
+                        // Blockist of items we never put in the baseline file.

```

Comment:
spell error.
```suggestion
                        // Blocklist of items we never put in the baseline file.
```

Reply:
It's already been fixed in a follow-up PR